<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Manish3150005" ignored="0" name="Test" tests="24" failures="10" timestamp="2021-04-18T23:54:52 IST" time="29.587" errors="0">
  <testcase name="checkUserCreation" time="2.494" classname="Assignment.ReqResTesting.CreateUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at Assignment.ReqResTesting.CreateUserTest.checkUserCreation(CreateUserTest.java:38)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkUserCreation -->
  <testcase name="checkUserCreation" time="1.757" classname="Assignment.ReqResTesting.CreateUserTest"/>
  <testcase name="checkUserCreation" time="1.141" classname="Assignment.ReqResTesting.CreateUserTest"/>
  <testcase name="checkUserCreation" time="1.36" classname="Assignment.ReqResTesting.CreateUserTest"/>
  <testcase name="checkUserCreationWithWrongBody" time="1.116" classname="Assignment.ReqResTesting.CreateUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:56)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkUserCreationWithWrongBody -->
  <testcase name="checkUserCreationWithWrongBody" time="0.831" classname="Assignment.ReqResTesting.CreateUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:56)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkUserCreationWithWrongBody -->
  <testcase name="checkUserCreationWithWrongBody" time="1.13" classname="Assignment.ReqResTesting.CreateUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:56)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkUserCreationWithWrongBody -->
  <testcase name="checkUserCreationWithWrongBody" time="1.241" classname="Assignment.ReqResTesting.CreateUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at Assignment.ReqResTesting.CreateUserTest.checkUserCreationWithWrongBody(CreateUserTest.java:56)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkUserCreationWithWrongBody -->
  <testcase name="deleteUser" time="1.226" classname="Assignment.ReqResTesting.DeleteUserTest"/>
  <testcase name="deleteUserWithWrongId" time="1.083" classname="Assignment.ReqResTesting.DeleteUserTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [204]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [204]
at Assignment.ReqResTesting.DeleteUserTest.deleteUserWithWrongId(DeleteUserTest.java:49)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserWithWrongId -->
  <testcase name="checkListOfusers" time="1.168" classname="Assignment.ReqResTesting.GetListOfUsersTest"/>
  <testcase name="checkListOfuserswithInvalidPage" time="1.144" classname="Assignment.ReqResTesting.GetListOfUsersTest">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at Assignment.ReqResTesting.GetListOfUsersTest.checkListOfuserswithInvalidPage(GetListOfUsersTest.java:59)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkListOfuserswithInvalidPage -->
  <testcase name="loginCheckForRegisteredUser" time="1.245" classname="Assignment.ReqResTesting.LoginTest"/>
  <testcase name="loginCheckForRegisteredUser" time="1.127" classname="Assignment.ReqResTesting.LoginTest"/>
  <testcase name="loginCheckForRegisteredUser" time="1.165" classname="Assignment.ReqResTesting.LoginTest"/>
  <testcase name="loginCheckForRegisteredUser" time="1.162" classname="Assignment.ReqResTesting.LoginTest"/>
  <testcase name="loginCheckForUnRegisteredUser" time="1.137" classname="Assignment.ReqResTesting.LoginTest"/>
  <testcase name="loginCheckWithWrongHttpMethod" time="0.657" classname="Assignment.ReqResTesting.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [405] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [405] but found [200]
at Assignment.ReqResTesting.LoginTest.loginCheckWithWrongHttpMethod(LoginTest.java:86)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginCheckWithWrongHttpMethod -->
  <testcase name="loginCheckWithWrongPassword" time="1.069" classname="Assignment.ReqResTesting.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [401] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [401] but found [200]
at Assignment.ReqResTesting.LoginTest.loginCheckWithWrongPassword(LoginTest.java:73)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginCheckWithWrongPassword -->
  <testcase name="loginCheckWithWrongHttpMethod" time="0.629" classname="Assignment.ReqResTesting.RegisterTest">
    <failure type="java.lang.AssertionError" message="expected [405] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [405] but found [200]
at Assignment.ReqResTesting.RegisterTest.loginCheckWithWrongHttpMethod(RegisterTest.java:73)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginCheckWithWrongHttpMethod -->
  <testcase name="m1" time="1.055" classname="Assignment.ReqResTesting.RegisterTest"/>
  <testcase name="m1" time="1.147" classname="Assignment.ReqResTesting.RegisterTest"/>
  <testcase name="m1" time="1.114" classname="Assignment.ReqResTesting.RegisterTest"/>
  <testcase name="m1" time="1.502" classname="Assignment.ReqResTesting.RegisterTest"/>
</testsuite> <!-- Test -->
